nixos/modules/services/development/vsmartcard-vpcd.nix

```diff
@@ -0,0 +1,41 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
```

Comment:
```suggestion
```
we should avoid adding new ones like that. Just prefix the relevant functions with lib.